packs/spell-effects/spell-effect-animal-form-ape.json

```diff
@@ -16,16 +16,16 @@
             "value": 2
         },
         "publication": {
-            "license": "OGL",
-            "remaster": false,
-            "title": "Pathfinder Core Rulebook"
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
         },
         "rules": [
             {
                 "key": "BattleForm",
                 "overrides": {
                     "senses": {
-                        "lowLightVision": {},
+                        "low-light-vision": {},
```

Comment:
Are we sure about this change? Generally `"parameter": "value"` is `"camelCase": "kebab-case"`. I know we just fixed the lingering camel cases in Sense RE data, but did that affect the parameter in BattleForm as well?

Reply:
Yes, I ran some tests. LLV won't apply if left in camelCase now on the battleforms. It only works if you use kebab-case (there's also a RE validation warning if it's left in camel).

Reply:
Hmm, if so that should have been migrated since some third parties use BattleForm. @stwlam thoughts?

Reply:
I'll fix that so the battle form class handles sluggifying

Reply:
Could this be left as-is so that it can serve as a confirmation that the future migration is working?

Reply:
Reverted the Ape effect back to camelCase for you.